makefiles/default-radio-settings.inc.mk

```diff
@@ -1,11 +1,25 @@
-# Set a custom channel if needed
-ifneq (,$(DEFAULT_CHANNEL))
+# ISM vairable can pass like: make BOARD=samr30-xpro ISM=EU flash term
+ifneq (EU,$(ISM))
+  # Default setting for US ISM band (Channel 26 for 2.4 GHz, Channel 5/Page 2 for sub-GHz)
+  ifneq (,$(DEFAULT_CHANNEL))
+    ifneq (,$(filter cc110x,$(USEMODULE)))        # radio is cc110x sub-GHz
+      CFLAGS += -DCC110X_DEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
+    endif
```

Comment:
This is not a 802.15.4 transceiver and might be better handled separate from the others. More specifically: The channels for the `cc110x` driver are virtual RIOT specific numbers. What they map to is determined by the channel map. The default channel map is EU only.

Reply:
Maybe my point is not clear. It is not possible to change the region by setting a different channel for that transceiver. Instead, one would have to fiddle with the parameters in `cc110x_params.h` to choose an appropriate channel map. In addition: While the chip is capable of supporting every base frequency between 300 MHz and 1GHz (with only the ISM bands in that range are tested by the vendor), the antenna circuit is specific to one ISM band. I don't know anyone with a CC110x connected to an antenna circuit suitable for 900 MHz. So no channel map for that was every tested or merged to RIOT. So someone in ITU region 2 would have to provide her/his own channel map as well; or better get that merged upstream.

Reply:
this PR is only meant to affect the AT86RF212B not the cc110x driver...
You may have a point about making it ITU region 1 rather than EU... we hand't thought of that.
Default is probably stuck on USA 915 ? as its been like this for a while? however if so few people use the samr30 then lets swap it now?
Cheers!

Reply:
we've worked out a sensible change - Thanks!

Reply:
Is the 868MHz SDR band actually used in the whole ITU region 1?

Reply:
looks like it: https://www.mapability.com/ei8ic/maps/regions.php

Reply:
Thanks for your suggestion! I made some changes , I hope it might be more sensible.

> One inline remark. Maybe it makes more sense to refer to the [ITU region](https://en.wikipedia.org/wiki/ITU_Region)? If I remember correctly, e.g. people in Africa are using the same channel maps as the EU (as both are region 1). It would be strange for them to select EU.
> 
> Also: Most users of RIOT are ITU region 1. So why not just making this the default?

Thanks for your suggestion! I made some changes , I hope it might be more sensible.

Reply:
Thanks for your suggestion! I made some changes , I hope it might be more sensible.

Reply:
we decided to propose FBAND for frequency band instead to avoid confusion

Reply:
You really should move the `cc110x` stuff out of this. E.g. when sticking to the `FBAND` approach it should be like this:

```Makefile
ifeq (868,$(FBAND))
  ...
else
  ...
endif

ifneq (,$(DEFAULT_CHANNEL))
  ifneq (,$(filter cc110x,$(USEMODULE)))        # radio is cc110x sub-GHz
    CFLAGS += -DCC110X_DEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
  endif
endif
```
As said: The ISM region cannot be chosen for the CC110x transceivers, it depends on the hardware (which antenna circuit is used). Thus, it is a hardware detail that should be better configured via `cc110x_params.h`.

Reply:
perhaps a PR for someone who can test a CC110x to make sure it tests ok?

@jue89 :
I like the suggestion to make a build-time error if an FBAND is not set - but would it cause auto build test issues? I have no idea how many people would be impacted by a change in the default for this radio..